healthcareai/common/transformers.py

```diff
@@ -1,8 +1,8 @@
 import numpy as np
 import pandas as pd
```

Comment:
This test works for an older version of pandas, for example the version of 0.19.2. But it doesn't work for the newest version like 0.20.1.